setup.py

```diff
@@ -42,6 +42,7 @@ def get_long_description():
         'numpy==1.13.1',
         'pandas==0.20.3',
         'numba==0.36.2',
+        'llvmlite==0.21.0',
```

Comment:
@RamanjaneyuluIdavalapati @prasannababuAddagiri Why do we need this new dependency? Is this for "interact" functionality?

Reply:
I am sorry Prashanth. We have mistaken. llvmlite is used by numba. When we install numba==0.36.2, then llvmlite==0.21.0 will automatically install. But on my machine, it didn't happen(the previous llvmlite==0.19.0 is used). I thought that it is problem with numba and we should manually install llvmlite==0.21.0 which is false. I will remove llvmlite.

Reply:
@RamanjaneyuluIdavalapati  No issues. Thank you.